posts/m&s_commits_to_digital_transformation_by_outsourcing_it_to_india.md

```diff
@@ -0,0 +1,19 @@
+# M&S Commits to Digital Transformation by Outsourcing it to India
+
+Stalwart of the British retail sector, M&S made a curious and conflicted announcement about their commitment to digital transformation last week.
+
+M&S have been taking heat for a number of years about their ability to adapt to the modern retail environment, from their [lambasted £150m e-commerce platform relaunch](https://www.marketingweek.com/2014/07/08/website-problems-hit-ms-sales/) that resulted in an 8% drop in sales, to their [lacklustre attempt at trialling grocery delivery](https://diginomica.com/2017/05/25/marks-spencers-lack-digital-urgency-costing-dearly/).
+
+The numbers have been equally disappointing, with their Christmas results [showing drops](http://www.iii.co.uk/articles/473924/ms-and-tesco-dive-christmas-results) in both food and clothing, the latter by nearly 3%. The market paints an even gloomier picture, with the M&S share price currently trading at a 25% discount on its 2017 high, and at nearly half of its 2015 ceiling.
+
+Likewise M&S has been struggling to hold on to its senior digital talent, with former Apple e-commerce head [Marcus East leaving after a year](https://www.retailgazette.co.uk/blog/2017/03/ms-digital-director-quits/) in the role early last year.
+
+This position makes last week's announcement that M&S' answer to digital transformation is to appoint Tata Consulting Services (TCS) to take care of it for them, all the more confusing.
+
+The [Press Release](http://www.retailtechnology.co.uk/news/6583/ms-commits-to-digital-transformation/) talks of agile methods, and pointedly reassures that transferring this capability to a consulting business best known for [delivering cost savings through offshoring](https://www.theregister.co.uk/2017/06/06/ey_plans_fresh_wave_of_indian_outsourcing/), is definitely the enabler to M&S becoming a "digital-first business".
+
+Unmentioned in the Press Release is that the deal results in 250 staff [shifting from M&S to TCS](http://www.telegraph.co.uk/business/2018/01/09/marks-spencer-ramps-cost-cutting-technology-overhaul/), with a further 30 roles expected to "dramatically change". M&S expects the move to shave £30m from its technology spend annually.
+
+Rather than recognising that true transformation needs to come from within the organisation through strong and savvy leadership, it's a real shame to see M&S take such a traditional and conservative approach to technology adoption. One that's seldom proved itself to be a real catalyst for success in the current fast-moving environment.
```

Comment:
The smallest of nitpicking, but I like this better with a dash, think it reads more punchily:

> Rather than recognising that true transformation needs to come from within the organisation through strong and savvy leadership, it's a real shame to see M&S take such a traditional and conservative approach to technology adoption — one that's seldom proved itself to be a real catalyst for success in the current fast-moving environment.